myStyles.css

```diff
@@ -0,0 +1,571 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@media (max-width: 768px) {
+  .generalPadding {
+    padding-left: 3%;
+    padding-right: 3%;
+  }
+
+  .mainContainer h1 {
+    font-size: 250%;
+    line-height: 39px;
+    padding-bottom: 5%;
+  }
+
+  .myLinks {
+    display: none;
+  }
+
+  .left {
+    display: flex;
+  }
+
+  .myLogo {
+    display: none;
+  }
+
+  .menuHamburgerContainer {
+    display: flex;
+    padding-top: 2%;
+  }
+
+  .textLinks {
+    display: none;
+  }
+
+  .importantLink {
+    display: none;
+  }
+
+  .main {
+    overflow: hidden;
+  }
+
+  .cardKind {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    padding-top: 3%;
+    padding-bottom: 3%;
+    background-color: #272a31;
+    margin-bottom: 10px;
+    margin-top: 10px;
+  }
+
+  .cardKind p {
+    max-width: 50%;
+  }
+
+  .cardKind h2 {
+    font-size: 20px;
+    width: 25%;
+  }
+
+  .cardKind img {
+    max-width: 10%;
+  }
+
+  .kinds hr {
+    width: 10%;
+    margin-bottom: 8%;
+  }
+
+  .kindsLkn {
+    display: none;
+  }
+
+  .footer {
+    display: none;
+  }
+
+  .textLinks a {
+    text-decoration: none;
+    color: rgb(251, 251, 251);
+    font-size: 45px;
+    padding-left: 3%;
+    padding-right: 3%;
+  }
+}
+
+@media (min-width: 769px) {
+  .generalPadding {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .mainContainer h1 {
+    font-size: 300%;
+    line-height: 45px;
+    padding-bottom: 2%;
+  }
+
+  .myLinks {
+    display: flex;
+  }
+
+  .left {
+    display: none;
+  }
+
+  .myLogo {
+    display: flex;
+  }
+
+  .menuHamburgerContainer {
+    display: none;
+  }
+
+  .menuTextContainer {
+    background-color: white;
+    display: flex;
+  }
+
+  .textLinks {
+    display: flex;
+  }
+
+  .importantLink {
+    display: flex;
+  }
+
+  .mainContainer p {
+    width: 70%;
+    margin-bottom: 3%;
+  }
+
+  .cardKind {
+    display: inline-flex;
+    flex-direction: column;
+    padding-top: 3%;
+    padding-bottom: 3%;
+    margin-bottom: 10px;
+    margin-top: 10px;
+    margin-left: 0.3%;
+    margin-right: 0.3%;
+    width: 25%;
+  }
+
+  .cardKind h2 {
+    text-align: center;
+    margin-top: auto;
+    margin-bottom: auto;
+  }
+
+  .kindsContainer {
+    display: flex;
+    justify-content: space-around;
+  }
+
+  .cardKind img {
+    max-width: 35%;
+    align-self: center;
+    margin-bottom: 6%;
+  }
+
+  .cardKind p {
+    margin-top: 8%;
+    margin-bottom: auto;
+    padding-left: 5%;
+    padding-right: 5%;
+    text-align: center;
+  }
+
+  .kinds hr {
+    width: 5%;
+    margin-bottom: 5%;
+  }
+
+  .kindsBtn {
+    display: none;
+  }
+
+  .bandBtn {
+    display: none;
+  }
+
+  .bandsContainer {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .bandKind {
+    flex: 1 0 50%;
+  }
+
+  .textLinks a {
+    text-decoration: none;
+    color: #272a31;
```

Comment:
Great job @alexansaa 🤝

- [ ] Great job implementing the changes requested by the previous reviewer. However, I can still see the `color: #272a31;` rule being repeated. Kindly create a single class to contain this rule and then use this class across your HTML code.